In [1]:
import numpy as np
from scipy.stats import norm

# Given data
sample_mean = 3050  # sample mean weekly cost (Rs. 3,050)
theoretical_mean = 1000 + 5 * 600  # theoretical mean weekly cost according to the cost model
sigma = 5 * 25  # standard deviation of the number of units produced in a week
n = 25  # sample size
alpha = 0.05  # significance level

# Step 1: State the Hypotheses statement:
# Null Hypothesis (H0): The mean weekly operating costs are as per the theoretical model.
# Alternative Hypothesis (H1): The mean weekly operating costs are higher than the theoretical model.

# Step 2: Calculate the Test Statistic
# Calculate the standard error of the sample mean
standard_error = sigma / np.sqrt(n)

# Calculate the test statistic (t)
t = (sample_mean - theoretical_mean) / standard_error

print("Test Statistic (t):", t)

# Step 3: Determine the Critical Value
# Calculate the critical value from the standard normal (Z) distribution table
critical_value = norm.ppf(1 - alpha)

print("Critical Value:", critical_value)

# Step 4: Make a Decision
# Compare the test statistic with the critical value to decide whether to reject the null hypothesis
if t > critical_value:
    print("Reject the null hypothesis. There is evidence to support the claim that the weekly operating costs are higher than the model suggests.")
else:
    print("Fail to reject the null hypothesis. There is no sufficient evidence to support the claim that the weekly operating costs are higher than the model suggests.")



Test Statistic (t): -38.0
Critical Value: 1.6448536269514722
Fail to reject the null hypothesis. There is no sufficient evidence to support the claim that the weekly operating costs are higher than the model suggests.


Based on the hypothesis testing process, where we compared the test statistic (t) of -38.0 with the critical value of 1.6448536269514722 at a significance level of 5%, we failed to reject the null hypothesis.

Therefore, we conclude that there is no sufficient evidence to support the restaurant owners' claim that the weekly operating costs are higher than the theoretical model suggests. This suggests that the observed mean weekly operating cost of Rs. 3,050 is not significantly higher than what would be expected based on the theoretical cost model, which predicts a mean weekly cost of 1,000 + 5X for X = 600 units.